inc/class-embed.php

```diff
@@ -18,7 +18,7 @@ public static function get( $id ) {
 			global $wpdb;
 
 			$row = $wpdb->get_row(
-				$wpdb->prepare( 'SELECT embed_id, src, embed_group_id, html FROM protected_embeds WHERE embed_id = %s', $id )
+				$wpdb->prepare( "SELECT embed_id, src, embed_group_id, html FROM {$wpdb->base_prefix}protected_embeds WHERE embed_id = %s", $id )
```

Comment:
Ahh we should probably bump the major version if we are changing the database table name, thinking Fusion are using this version currently.